sdl2-hyper-sonic-drivers/test/HyperSonicDrivers/files/westwood/TestADLFile.cpp

```diff
@@ -56,7 +56,7 @@ namespace HyperSonicDrivers::files::westwood
         ADLFileMock f("../fixtures/DUNE0.ADL");
 
         EXPECT_EQ(f.getVersion(), 2);
-        EXPECT_EQ(f.getNumTracks(), 18);
+        EXPECT_EQ(f.getNumTracks(), 200);
```

Comment:
another program reported 200 tracks. so i am not reading exactly right. in fact i am missing sounds.... so here we go.

Reply:
not sure neither 200 tracks is right.. but the "hidden" sounds are played there, even though there are at least half empty tracks.